### PR TITLE
Improve help for creating / being added to new projects

### PR DESCRIPTION
**Pull request by liggitt**
Updates help text shown on the project page to describe how to create a new project or add the current user to an existing project

### CODE REVIEW

**Comment by liggitt** [comment]
@jwforres @deads2k

**Comment by liggitt** [comment]
20150220_122052_empty_projects

File: assets/app/views/projects.html [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-    <div ng-if="(projects | hashSize) == 0">
-      <div ng-bind-html="emptyMessage"></div>
-    </div>
     <div class="tile tile-click" ng-click="tileClickHandler($event)" style="margin-bottom: 10px;" ng-repeat="project in projects">
       <h2><a class="tile-target" href="/project/{{project.metadata.name}}">{{project.displayName || project.metadata.name}}</a></h2>
       <div class="muted" style="margin-top: -5px;" ng-if="project | annotation : 'description'">{{project | annotation : 'description'}}</div>
     </div>
+    <div ng-if="emptyMessage && (projects | hashSize) == 0">{{emptyMessage}}</div>
+    <div style="margin-top: 10px;">
+      To create a new project, run <code>openshift ex new-project &lt;projectname&gt; --admin={{user.metadata.name || '&lt;YourUsername&gt;'}}</code>
```
Comment (deads2k) [inline assets/app/views/projects.html]: is `user.metadata.name` scope qualified?

Comment (liggitt) [inline assets/app/views/projects.html]: user is the current user

**Comment by deads2k** [comment]
lgtm

**Comment by jwforres** [comment]
LGTM

**Comment by liggitt** [comment]
[merge]

**Pull request merged**